instrumentation/wicket-6.0/javaagent/wicket-6.0-javaagent.gradle

```diff
@@ -0,0 +1,21 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = 'org.apache.wicket'
+    module = 'wicket'
+    versions = "[6.0.0,]"
+    assertInverse = true
```

Comment:
I don't think this line is needed. @trask @mateuszrzeszutek Do we have any guidelines, when to use inverse assert and when not?

Reply:
I don't think that we have any "official" recommendation, but I remember that in some discussion somebody has said that "it should have been turned on by default" - and I can agree with that.
If there's some old library version that we know it can't be instrumented properly, but muzzle reference check does not block it from being applied, then `assertInverse` forces the author of the instrumentation module to use `classLoaderMatcher()` to make sure that this old version won't be instrumented (and thus won't break the instrumented application).

Reply:
I thought that `assertInverse` is used to ensure that version range starts with earliest possible version where muzzle checks pass. For example adding `assertInverse = true` to struts-2.3 would fail `./gradlew :instrumentation:struts-2.3:javaagent:muzzle`

Reply:
> For example adding assertInverse = true to struts-2.3 would fail ./gradlew :instrumentation:struts-2.3:javaagent:muzzle

Then it would be a good idea to introduce a `classLoaderMatcher()` to struts-2.3; or take a look at the instrumentation and try yo determine why it does not work for earlier versions.

Anyway, since usage patterns of `assertInverse` seem to be a bit unclear, we should improve the muzzle docs - I'll take care of it.

Reply:
Instrumentation probably does work, test might not. Muzzle fails with MUZZLE PASSED Struts2InstrumentationModule BUT FAILURE WAS EXPECTED. Considering Struts 2.3.1 is dated 08-Dec-2011 not bothering with earlier versions is probably a deliberate choice.